iquip/apps/builder.py

```diff
@@ -291,13 +290,47 @@ def value(self) -> Optional[str]:
         return None
 
 
+class _ScanEntry(QWidget):
+    """Entry class for a Scannable object.
+    
+    Attributes:
+        name: The name of the scannable object.
+        argInfo: Each key and its value are:
+          default: The Dictionary that describes arguments of a specific scanning type.
+          unit: The unit of the number value.
+          scale: The scale factor that is multiplied to the number value.
+          global_step: The step between values changed by the up and down button.
+          global_min: The minimum value. (default=float("-inf"))
+          global_max: The maximum value. (default=float("inf"))
+            If min > max, then they are swapped.
+          ndecimals: The number of displayed decimals.
+    """
+    def __init__(
+        self,
+        name: str,
+        argInfo: Dict[str, Any],
+        parent: Optional[QWidget] = None
+        ):
+        """Extended.
+
+        Args:
+            name: See the attributes section.
+            argInfo: See the attributes section.
+        """
+        super().__init__(parent=parent)
+        #TODO(AIJUH): Add feature for argInfo processing and other Scan type classes.
```

Comment:
Please write "# TODO" with a blank.

Reply:
Also, I think you need to remove detailed docstring of `argInfo` like below since it will be invalid.
```
"""
 Attributes:
        name: The name of the scannable object.
        argInfo: The infomation of the arguments.
"""
```
Or you may remove them from attributes and keep them only in `__init__()` arguments.

And please move the TODO comment to the above of the class definition.